pkgs/applications/networking/instant-messengers/linphone/default.nix

```diff
@@ -18,6 +18,12 @@ stdenv.mkDerivation rec {
     sha256 = "0az2ywrpx11sqfb4s4r2v726avcjf4k15bvrqj7xvhz7hdndmh0j";
   };
 
+  cmakeFlags = "-DENABLE_GTK_UI=ON";
+
+  postPatch = ''
+    touch coreapi/liblinphone_gitversion.h
```

Comment:
Why is this necessary? What does it do?

Reply:
The source file `gtk/main.c` depends on this file.  Linphone actually has two build systems, one based on cmake, the other one based on autoconf/automake.  The `coreapi/Makefile.am` file actually provides the `liblinphone_gitversion.h` file (by just touching it), however cmake doesn't.  Since otherwise the build would now fail I just touch the file (like `coreapi/Makefile.am` would)

Reply:
That sounds like a bug, can you please open an upstream ticket for that and link it in a comment?